### ISSUE TITLE
Add setting for read-only database open mode

### ISSUE THREAD

**Issue by dancarroll** [TheAngryByrd/MetroPass#52]
I'd like a setting to set the app to read-only mode, so that it does not write back to the database file.

Today, when trying out the app, I noticed a couple of issues:
1. After playing around copying/pasting usernames and passwords, snap views, and testing out whether the clipboard clear feature worked (did not seem like it ever cleared the clipboard), I noticed that the passwords became corrupted (they no longer matched the original passwords in the database)
2. Trying to reopen the database on my desktop computer, the database was no longer compatible with KeePass.

To alleviate both of those issues, it would be nice to prevent MetroPass from modifying the database file.

Let me know if you want further details on problem 1 I ran into.  Not opening another issue right now, since I don't have any good debugging steps at the moment.

**Comment by TheAngryByrd** [comment]
Oh wow. That's not good.  I'm real sorry for that :(.  I hope you had a backup.  I wonder what stupid piece of code did that.  

Yeah that's definitely a good idea setting MetroPass to read-only mode then enabling it in the settings.

Bugs like this are going to be hard to figure out since I'm not able to use someone else's database for obvious reasons.  If you can somehow create a problem with a sample database I would love an example of the problem.  I know early versions of KeePass 2.x databases were built slightly different and maybe I'm not accounting for it  Knowing what version you created it with originally and other modifications is a pretty daunting task to recreate.

**Comment by dancarroll** [comment]
Yes, luckily I was working off a copy I put on Skydrive, so my main database was not affected (phew!).

I had been using an old version of Keepass (2.13, I believe). I upgraded to the latest version and created a fresh copy  of the DB to try again. For the time being, I have disabled the clipboard auto-clear, since it did not seem to work at all, and I wanted to get back to default settings.

I'll see if I can hit the corruption problem again and try to get some better repro steps.

In the meantime, thanks for the app. It is great to get any access to my Keepass database on Windows RT, let alone something that looks good and interacts well in snap view with other apps.
